feat: detect javascript methods import fetch console.log `console.dir`

EsadCetiner opened the pull request:
Blocks four new JavaScript methods and can now detect JavaScript injection attacks with curly brackets.
```
import
fetch
console.log
console.dir
```

Although this PR introduces one new false positive detected via `ftw quantitative`:
`If you’re looking for a personal recommendation, if you’re willing to import (and perhaps wait around, as they often sell out line has been consistently excellent so far.`

closes #3632
closes: #3633

Xhoenix commented on regex-assembly/941390.ra:
```diff
@@ -14,3 +14,7 @@ atob
 confirm
+import
+fetch
+console.log
+console.dir
```
Escape the `.`

Xhoenix commented:
`import` and `fetch` are very commonly used words in natural language, and are very likely to cause false positives. This needs to be discussed.

EsadCetiner commented:
I've added it to the agenda but I'll likely won't be in the CRS chat so I'll leave my two cents in this comment.

Adding `import` and `fetch` should not cause many false positives, there's only really one known false positive relating to `import`. The same could be said about the other methods in the rule, and there haven't been any reports on those methods causing false positives nor have I encountered any personally.

Xhoenix commented:
Yeah, I understand the suffix is there to prevent false positives, but it's still better if we discuss this, or if someone else takes a look on this, and approves. 😄

theseion reviewed (approved):
Awesome! ❤️

theseion commented:
I think the risk of FPs is acceptable.

Xhoenix commented:
**Note**: There aren't any tests for fetch, console.log and console.dir.

EsadCetiner commented:
@Xhoenix I think it's fine since I'm just adding methods to a list. The first test I added was a known false positive, maybe somebody can find a fix for it in the future that I couldn't find, and the second one was added because I modified the suffix for the rule.

I'd rather not bloat the tests file with largely useless tests, sometimes less is more.
